lib/runtime-core/src/backing.rs

```diff
@@ -17,30 +17,31 @@ use crate::{
 };
 use std::slice;
 
+/// The `LocalBacking` "owns" the memory used by all the local resources.
+/// That is, local memories, tables, and globals (as well as some additional
+/// data for the virtual call machinery).
```

Comment:
One suggestion
> The `LocalBacking` "owns" the state used by

Could you clarify if LocalBacking belongs to an instance or module?